subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1565,6 +1573,16 @@ static uint8_t vs_cmd_put(uint8_t const * const cmd,
 #ifdef CONFIG_BT_CENTRAL
 	case SDC_HCI_OPCODE_CMD_VS_CENTRAL_ACL_EVENT_SPACING_SET:
 		return sdc_hci_cmd_vs_central_acl_event_spacing_set((void *)cmd_params);
+#endif
+#if defined(CONFIG_BT_CTLR_SDC_EVENT_TRIGGER)
+	case SDC_HCI_OPCODE_CMD_VS_SET_CONN_EVENT_TRIGGER:
+		return sdc_hci_cmd_vs_set_conn_event_trigger((void *)cmd_params);
+#endif
+#if defined(CONFIG_BT_CONN)
+	case SDC_HCI_OPCODE_CMD_VS_GET_NEXT_CONN_EVENT_COUNTER:
```

Comment:
can this be used for anything else than conn_event_trigger? I wander if it should go under CONFIG_BT_CTLR_SDC_EVENT_TRIGGER as well

Reply:
I'm not sure. Never really know what a customer might come up with. This command is also very small